File: 2018 czerwiec/python/1.py
with open("dane1.txt", "r") as f:
    d1 = f.read().strip().split('\n')

with open("dane2.txt", "r") as f:
    d2 = f.read().strip().split('\n')

ans = 0
with open("wynik4_1.txt", "w") as f:
    for l1, l2 in zip(d1, d2):
        l1 = list(map(int, l1.split()))
        l2 = list(map(int, l2.split()))
        if l1[-1] == l2[-1]:
            ans += 1
    f.write(str(ans))

File: 2018 czerwiec/c++/1.cpp
#include <iostream>
#include <fstream>
using namespace std;

int dane1[1005][15], dane2[1005][15];

int main()
{
    ifstream in1("dane1.txt");
    ifstream in2("dane2.txt");
    ofstream out("wynik4_1.txt");

    int ans = 0;
    for (int i = 1; i <= 1000; i++)
    {
        for (int j = 1; j <= 10; j++)
        {
            in1 >> dane1[i][j];
            in2 >> dane2[i][j];

            if (j == 10 && dane1[i][j] == dane2[i][j])
                ans++;
        }
    }
    out << ans;
}